aas_core_meta/v3.py

```diff
@@ -1,5 +1,11 @@
 """
-Provide the meta-model for Asset Administration Shell V3.0.
+The presented version of the Metamodel within this documentation
+is related to the work of aas-core-works, which can be found here.
+The presented content of this website is neither related to the IDTA nor
```

Comment:
The presented content of this documentation is no official release of the IDTA or

Reply:
"The presented content is no official ..."

Docstring is going to be translated to SDK code, HTML etc. The term "documentation" might be misleading in many contexts.

Reply:
Please also remove any mentions of "website" as this is misleading in context of SDKs etc.

Reply:
See 59c1702a0ef8809115c3eaa8823a0a1dc3435726

Reply:
Thanks, LGTM!